home/home2.2/image.java

```
@@ -0,0 +1,184 @@
+                if (SplitedLine.length!= Constants.MatrixSize){
+                    return Constants.sizeProblem;
+                }       
+                for (int i = 0; i <SplitedLine.length; i++) {
+                    Pixel pix=new Pixel(SplitedLine[i]);
+                    if (!pix.isValid()) {
+                        return Constants.colorProblem;
+                    }
+                    pixels[i]=pix;
+                }
+                if (indexCounter == Constants.MatrixSize) {
+                    return Constants.sizeProblem;
+                }
+                lines[indexCounter]=pixels;
+                indexCounter++;
+            }    
+        } catch (Exception e) {}
+        return Constants.Fine;
+    }
+
+    public void changeMatrix(Pixel favouriteColor, Pixel hatedColor){
+        for (int i = 0; i < Constants.MatrixSize; i++) {
+            for (int j = 0; j < Constants.MatrixSize; j++) {
+                if (lines[i][j].compareColor(favouriteColor)) {
+                    if (i > 0) {
+                        lines[i - 1][j].setColor(favouriteColor);
+                    }
+                    if (j > 0) {
+                        lines[i][j - 1].setColor(favouriteColor);
+                    }
+                }
+                if (lines[i][j].compareColor(hatedColor) && hatedColor.isValid()) {
+                    lines[i][j].setColor(favouriteColor);    
+                }
+            }
+        }
+    }
+    
+    public void saveFile() {
+    try (BufferedWriter fileEdit = new BufferedWriter(new FileWriter("Output" + pixelMatrixFile))) {
+        for (int i = 0; i < Constants.MatrixSize; i++) {
+            String lineStr = arrayToString(lines[i]);
+            if (!lineStr.isEmpty()) {
+                fileEdit.write(lineStr);
+                if (i < Constants.MatrixSize - 1) {
+                    fileEdit.newLine();
+                }
+            }
+        }
+    } catch (Exception e) {}
```

empty catch looks a bit suspicious